LEAF_Request_Portal/js/gridInput.js

```diff
@@ -11,6 +11,18 @@ var gridInput = function(gridParameters, indicatorID, series, recordID) {
         dropdownElement += '</select>';
         return dropdownElement;
     }
+    function makeMultiselect(options, selected){
```

Comment:
Need at least "options" given a default value to make it an array/list.  The "options.length" function can cause odd issues depending on the browser and the version of js it's using.  Besides, we need to get in the habit of providing default values for pretty much everything so we can start enforcing "type".

Reply:
I'm a little confused, options is a parameter here being passed from where the function is called down at line 72/75

So I figured I wouldn't want to redefine it if its already being passed into this function.

Reply:
Yes.  But generally these days with functions and methods being reused every where, you want to give them default values in the function call to prevent differing behavior.  In this case, it would be something like "function makeMultiselect(options=array(), selected=FALSE){..."   They would get over written if the values are passed in, but keep the function from "breaking" if they are missed or lost for some reason.

Reply:
Understood thanks for the comment.